Adds new icons for FoW

origami-z opened the pull request:
TODO:
- [x] decide whether success solid is a breaking change or not 

Adds following icons
- Accessible
- Cart
- Crops
- Github
- Hospital
- Man
- Man woman
- Map
- Marker
- Medical kit
- Picnic
- Pot food
- Signpost
- Stethoscope
- Storefront
- Success
- Tote
- Universal access
- Utensils
- Woman
- Woodland

Closes #1276

origami-z commented:
@pseys success-fill changed completely, which will be a breaking change...?

origami-z commented on packages/icons/src/SVG/success-solid.svg:
This is a breaking change needs discussion

pseys replied:
Is there any way for us to know if success-solid is being used anywhere currently? Its not included in any of our components.

origami-z replied:
@joshwooding do we treat this as breaking change?

joshwooding replied:
Hmm, I would say no but I think it's up to design whether it's a breaking change or not. The change is drastic but I can't remember how visual changes are being treated cc @bhoppers2008

bhoppers2008 replied:
I think we can avoid the discussion around what constitutes a breaking change here. Given success-solid looks the same as success tick looks the same as success, let's class this is a design bug that we're rectifying and go ahead and merge.

pseys commented:
@origami-z I'm not sure why but medical-kit-solid isn't displaying correctly in Storybook. It seems fine in Figma.

origami-z commented:
> @origami-z I'm not sure why but medical-kit-solid isn't displaying correctly in Storybook. It seems fine in Figma.

Yes, i will re-check everything again.

origami-z commented:
Checked everything
- removed stroke color on medical-kit-solid in Figma
- remove clip content on component layer setting to avoid unnecessary `clipPath` introduced (also added to the salt icon contribution checker plugin)
@pseys
